packages/template-chakra-storefront/src/pages/product-detail/index.test.js

```diff
@@ -25,18 +25,7 @@ import {
     basketWithProductBundle,
     bundleProductItemsForPDP
 } from '../../../mocks/product-bundle'
-
-const mockAddToWishlist = jest.fn()
-const mockIsItemInWishlist = jest.fn().mockReturnValue(false)
-jest.mock('../../hooks/use-wish-list', () => ({
```

Comment:
Replacing hook level mock with api level mock. Since this is tests for page components, we should aim to simulate what the shoppers do on the browser and assert what is happening on the browser rather then check if a func is called. This will give us a certain level of confident when working with Page level components

Reply:
Thanks for taking the extra effort to do that 🙏